app/handlers/retrieveResults.py

```diff
@@ -16,5 +21,32 @@ class RetrieveResultsHandler(tornado.web.RequestHandler):
 
     @apiError error_message The error message to display.
     """
+
+    def post(self):
```

Comment:
Should retrieve results have a post function? If so, it should do something like returning a helpful error message